src/Microsoft.Identity.Client/CacheV2/Schema/Credential.cs

```diff
@@ -229,18 +230,18 @@ public override string ToString()
         {
             var sb = new StringBuilder();
             sb.Append("Credential(\n");
-            sb.Append($"  HomeAccountId: {HomeAccountId}\n");
-            sb.Append($"  Environment: {Environment}\n");
-            sb.Append($"  Realm: {Realm}\n");
-            sb.Append($"  CredentialType: {CredentialType}\n");
-            sb.Append($"  ClientId: {ClientId}\n");
-            sb.Append($"  FamilyId: {FamilyId}\n");
-            sb.Append($"  Target: {Target}\n");
-            sb.Append($"  CachedAt: {CachedAt}\n");
-            sb.Append($"  ExpiresOn: {ExpiresOn}\n");
-            sb.Append($"  ExtendedExpiresOn: {ExtendedExpiresOn}\n");
-            sb.Append($"  Secret: {Secret}\n");
-            sb.Append($"  AdditionalFieldsJson: {AdditionalFieldsJson}\n");
+            sb.Append(string.Format(CultureInfo.InvariantCulture, "  HomeAccountId: {0}\n", HomeAccountId));
+            sb.Append(string.Format(CultureInfo.InvariantCulture, "  Environment: {0}\n", Environment));
+            sb.Append(string.Format(CultureInfo.InvariantCulture, "  Realm: {0}\n", Realm));
+            sb.Append(string.Format(CultureInfo.InvariantCulture, "  CredentialType: {0}\n", CredentialType));
```

Comment:
Isn't there AppendFormat on stringbuilder that takes a cultureinfo?

Reply:
Yes, I didn't klnow this when writing the code.